libs/test-utils/src/jwt/generate-keys-and-jwt.ts

```diff
@@ -0,0 +1,67 @@
+import type { JWTPayload, CryptoKey } from 'jose';
+import {
+  generateKeyPair,
+  SignJWT,
+  calculateJwkThumbprint,
+  exportJWK,
+} from 'jose';
+
+interface GenerateJwtPayload {
+  issuer: string;
+  audience: string | string[];
+  jti: string;
+  payload: JWTPayload | undefined;
+  alg: string;
+  expiryDate: Date;
+  typ?: string;
+  kid?: string;
+}
+
+interface GeneratedJwtResponse {
+  privateKey: CryptoKey;
+  publicKey: CryptoKey;
+  publicKid: string;
+  jwt: string;
+}
+
+const signWithPrivateKey = async (
```

Comment:
Looks good! I think there's a few test helpers we can share these methods with also now 🚀